### PR TITLE
www/nginx VTS frontend

### PR DESCRIPTION
**Pull request by fabianfrz**
@fichtner I took the original one from the project repository and refactored it to more modern JS (the tag generator function is still there) and integrated it into the plugin (translation and upstream name translation for better usability)

### CODE REVIEW

File: www/nginx/src/opnsense/www/css/nginx/vts.css [hunk 1]
```diff
@@ -0,0 +1,329 @@
+/*
+* Copyright (C) 2017-2018 Fabian Franz
+* Copyright (C) 2015, YoungJoo.Kim <vozltx@gmail.com>
```
Comment (fichtner) [inline www/nginx/src/opnsense/www/css/nginx/vts.css]:
```suggestion
* Copyright (C) 2015 YoungJoo.Kim <vozltx@gmail.com>
```

File: www/nginx/src/opnsense/mvc/app/views/OPNsense/Nginx/vts.volt [hunk 2]
```diff
@@ -0,0 +1,477 @@
+{#
+# Copyright (C) 2017-2018 Fabian Franz
+# Copyright (C) 2015, YoungJoo.Kim <vozltx@gmail.com>
```
Comment (fichtner) [inline www/nginx/src/opnsense/mvc/app/views/OPNsense/Nginx/vts.volt]:
```suggestion
# Copyright (C) 2015 YoungJoo.Kim <vozltx@gmail.com>
```

**Comment by fichtner** [comment]
doesn't this need correction in pkg-descr ?

**Comment by fabianfrz** [comment]
@fichtner the release note already exists: https://github.com/opnsense/plugins/blob/master/www/nginx/pkg-descr#L13

**Pull request merged**